test/IntegrationTests/MSTest.Acceptance.IntegrationTests/TestContextTests.cs

```diff
@@ -34,6 +34,16 @@ public async Task TestContext_TestData_PropertyContainsExpectedValue()
         testHostResult.AssertOutputContainsSummary(failed: 0, passed: 3, skipped: 0);
     }
 
+    public async Task TestContext_TestException_PropertyContainsExpectedValue()
+    {
+        var testHost = TestHost.LocateFrom(_testAssetFixture.ProjectPath, TestAssetFixture.ProjectName, TargetFrameworks.NetCurrent.Arguments);
+        TestHostResult testHostResult = await testHost.ExecuteAsync("--filter ClassName~TestContextException");
+
+        // Assert
+        testHostResult.AssertExitCodeIs(2);
+        testHostResult.AssertOutputContainsSummary(failed: 2, passed: 1, skipped: 0);
```

Comment:
Do we want to assert exactly that the two failures are `TestContextExceptionFailingInTestInit` and `TestContextExceptionFailingInTestMethod` while `TestContextExceptionNotFailing` passes?